### PR TITLE
[API-94] Send the UA header, along with the SDK name and version, on every API request

### PR DESCRIPTION
**Pull request by betacar**

### CODE REVIEW

File: tests/login.test.js [hunk 1]
```diff
@@ -70,3 +72,52 @@ describe("Initialize Bynder with permanent token", () => {
...
+
+
+describe("API call headers", () => {
+  describe("with permanent token", () => {
+    let mockedFunction = null;
+
+    beforeEach(() => {
+      bynder = new Bynder(configs);
+      bynder.api.permanentToken = "token";
+      mockedFunction = bynder.api._axios;
```
Comment (githuib) [inline tests/login.test.js:84]: I think another interesting strategy could be to mock the `axios` module, like described here: https://jestjs.io/docs/en/mock-functions#mocking-modules (coincidentally they have axios in their example as well :) )

That way, you don't have to alter the application code (with `this._axios`).

**Review by githuib — COMMENTED** [review]
Nice work! My only point would be that we can probably simplify the mocking strategy a bit, see my comment in the code.

File: tests/login.test.js (hunk 1 above)
Comment (betacar) [reply tests/login.test.js]: Restored.

File: tests/login.test.js [hunk 2]
```diff
@@ -70,3 +75,48 @@ describe("Initialize Bynder with permanent token", () => {
...
+    beforeEach(() => {
+      bynder = new Bynder(configs);
+      axios.mockResolvedValue({});
+      bynder.api.permanentToken = "token";
+    });
+
+    afterEach(() => {
+      axios.mockRestore();
+      bynder = null;
+    });
```
Comment (githuib) [inline tests/login.test.js:91]: Are these having any effect? Since they should be reset in the `beforeEach` before each test.

Comment (betacar) [reply tests/login.test.js:91]: Nope. They're being reset on this method after each test.

Comment (githuib) [reply tests/login.test.js:91]: Yes, but also before each test we do:
```
      bynder = new Bynder(configs);
      axios.mockResolvedValue({});
      bynder.api.permanentToken = "token";
```

So I'd say at least the `bynder = null;` wouldn't have any effect

**Review by githuib — APPROVED** [review]
Don't want to block it because of that comment, since that code shouldn't harm. Mainly trying to understand what's going on and if I missed something 👍

**Pull request merged**